### PR TITLE
Initialize repository

### PR DESCRIPTION
**Pull request by tfussell**
Towards https://github.com/giantswarm/giantswarm/issues/20281

### CODE REVIEW

File: helm/default-apps-openstack/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: 0.0.1
+name: default-apps-openstack
+description: A Helm chart for default-apps-openstack
+engine: gotpl
+home: https://github.com/giantswarm/default-apps-openstack-app
+sources:
+- https://github.com/some-org/some-repo
+version: 0.1.0
+annotations:
```
Comment (tfussell) [inline helm/default-apps-openstack/Chart.yaml:10]: I don't know what these are.

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
       - architect/push-to-app-catalog:
           context: "architect"
-          # executor: "app-build-suite" # uncomment this if you want automatic metadata generation and helm chart linting
-          name: "package and push {APP-NAME} chart"
-          app_catalog: "giantswarm-playground-catalog"
-          app_catalog_test: "giantswarm-playground-test-catalog"
-          chart: "{APP-NAME}"
+          executor: "app-build-suite"
+          name: "package and push default-apps-openstack chart"
+          app_catalog: "default-catalog"
```
Comment (tfussell) [inline .circleci/config.yml:12]: @giantswarm/team-honeybadger Which catalog should this go in?

File: helm/default-apps-openstack/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  branch: my-branch
+  commit: f00bar
```
Comment (rossf7) [inline helm/default-apps-openstack/Chart.yaml:12]:
```suggestion
annotations:
  application.giantswarm.io/team: "honeybadger"
```
We use annotations for routing [app alerts](https://intranet.giantswarm.io/docs/dev-and-releng/app-developer-processes/routing_app_alerts/). Any annotations added here also get added to the AppCatalogEntry CRs.

File: helm/default-apps-openstack/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```
Comment (rossf7) [inline helm/default-apps-openstack/Chart.yaml:1]:
```suggestion
apiVersion: v2
```

File: helm/default-apps-openstack/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+appVersion: 0.0.1
+name: default-apps-openstack
+description: A Helm chart for default-apps-openstack
+engine: gotpl
+home: https://github.com/giantswarm/default-apps-openstack-app
+sources:
+- https://github.com/some-org/some-repo
```
Comment (rossf7) [inline helm/default-apps-openstack/Chart.yaml:8]:
```suggestion
```

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -1,33 +1,17 @@
...
-  - [ ] AWS
-  - [ ] Azure
-  - [ ] KVM
 
 #### Other testing
 
-Description of features to additionally test for {APP-NAME} installations.
+Description of features to additionally test for default-apps-openstack installations.
 
 - [ ] check reconciliation of existing resources after upgrading
```
Comment (MarcelMue) [inline .github/pull_request_template.md:16]: Let's remove these checkboxes as well. Do not see the purpose in this project.

**Review by MarcelMue — APPROVED** [review]

File: .github/pull_request_template.md (hunk 6 above)
Comment (tfussell) [reply .github/pull_request_template.md]: Done.

**Pull request merged**